### ISSUE TITLE
Basic demo fails with DomainError()

### ISSUE THREAD

**Issue by Nectarineimp** [panlanfeng/KernelEstimator.jl#5]
xdata = randn(1000)
kerneldensity(xdata)
DomainError:
 in nan_dom_err at .\math.jl:196 [inlined]
 in log(::Float64) at .\math.jl:202
 in gaussiankernel(::Float64, ::Array{Float64,1}, ::Float64, ::Array{Float64,1}, ::Int64) at~\.julia\v0.5\KernelEstimator\src\kernel.jl:132
 in #kerneldensity#11(::Array{Float64,1}, ::Float64, ::Float64, ::KernelEstimator.#gaussiankernel, ::Float64, ::Function, ::Array{Float64,1}) at ~\.julia\v0.5\KernelEstimator\src\density.jl:12
 in kerneldensity(::Array{Float64,1}) at ~\.julia\v0.5\KernelEstimator\src\density.jl:4
 in include_string(::String, ::String) at .\loading.jl:441
 in eval(::Module, ::Any) at .\boot.jl:234
 in (::Atom.##67#70)() at ~\.julia\v0.5\Atom\src\eval.jl:40
 in withpath(::Atom.##67#70, ::Void) at ~\.julia\v0.5\CodeTools\src\utils.jl:30
 in withpath(::Function, ::Void) at ~\v0.5\Atom\src\eval.jl:46
 in macro expansion at ~\.julia\v0.5\Atom\src\eval.jl:109 [inlined]
 in (::Atom.##66#69)() at .\task.jl:60

**Comment by panlanfeng** [comment]
Unable to reproduce it. This happens when bandwidth is negative. This happened before but it was already fixed.
Have you tried to update the package?

**Comment by Nectarineimp** [comment]
I ran it from the REPL in Atom.io on a Windows machine whose uptime was over two weeks... I'm rebooting it and seeing if that clears up something. I did Pkg.update() so I don't think that was the issue.OK one reboot later... and it's working. I hate Windows. Sorry for the trouble.
On February 15, 2017 at 4:50 PM Lanfeng Pan <notifications@github.com> wrote:Unable to reproduce it. This happens when bandwidth is negative. This happened before but it was already fixed.
Have you tried to update the package?
